src/server.c

```diff
@@ -5330,7 +5376,9 @@ sds genRedisInfoString(const char *section) {
             "total_reads_processed:%lld\r\n"
             "total_writes_processed:%lld\r\n"
             "io_threaded_reads_processed:%lld\r\n"
-            "io_threaded_writes_processed:%lld\r\n",
+            "io_threaded_writes_processed:%lld\r\n"
+            "reply_buffer_shrink:%lld\r\n"
```

Comment:
please list all the interface changes in the PR's top comment (the ones for INFO and for CLIENT LIST)
i.e. add a section about interface changes.

Reply:
Sure - will be done once I push the next changes

Reply:
@oranagra I edited the PR comment. take a look when you have the time and tell me if it is well explained.